test/Base_Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -119,6 +120,18 @@ spec_with suite_builder name create_new_datetime parse_datetime nanoseconds_loss
             text = create_new_datetime 1970 (zone = Time_Zone.utc) . to_text
             text . should_equal "1970-01-01 00:00:00Z[UTC]"
 
+        group_builder.specify "should convert to Enso code" <|
+            create_new_datetime 1970 . pretty . should_equal "Date_Time.new 1970 1 1"
+            create_new_datetime 1923 9 24 . pretty . should_equal "Date_Time.new 1923 9 24"
+            create_new_datetime 1923 9 24 12 20 44 . pretty . should_equal "Date_Time.new 1923 9 24 12 20 44"
```

Comment:
I'd appreciate a test where hours and minutes are 0, so that we get the `seconds=`